extensions/utils/VaultClientWrapper.js

```diff
@@ -21,7 +21,7 @@ class VaultClientWrapper {
     }
 
     init() {
-        if (this._authConfig.type !== authTypeAssumeRole) {
+        if (![authTypeAssumeRole, authTypeNone].includes(this._authConfig.type)) {
```

Comment:
`[authTypeAssumeRole, authTypeNone]`
Can this be a variable somewhere ? To avoid repetition and ease the maintenance in case we want to add one for some reason?

Reply:
Those are already constants